Subject: cpumf: correct z14 counter number
From: Hendrik Brueckner <brueckner@linux.ibm.com>

Summary:     cpumf: Add CPU-MF hardware counters for z14
Description: Add hardware counter definitions for IBM z14.
Upstream-ID: 144bddbf5bce749549a289acbeb49337edaaea45
Problem-ID:  KRN1608

Upstream-Description:

             cpumf: correct z14 counter number

             Signed-off-by: Hendrik Brueckner <brueckner@linux.vnet.ibm.com>
             Signed-off-by: Stefan Haberland <sth@linux.vnet.ibm.com>


Signed-off-by: Hendrik Brueckner <brueckner@linux.ibm.com>
---
 cpumf/data/cpum-cf-extended-z14.ctr |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/cpumf/data/cpum-cf-extended-z14.ctr
+++ b/cpumf/data/cpum-cf-extended-z14.ctr
@@ -269,7 +269,7 @@ Description:
 Decimal instructions dispatched. Instructions: CVB, CVD, AP, CP, DP, ED,
 EDMK, MP, SRP, SP, ZAP
 .
-Counter:233	Name:LAST_HOST_TRANSLATIONS
+Counter:232	Name:LAST_HOST_TRANSLATIONS
 Description:
 Last Host Translation done
 .
